packages/server/integrations/helpers/authorizeOAuth2.ts

```diff
@@ -10,6 +10,29 @@ interface AuthorizeOAuth2Params {
   additonalHeaders?: Record<string, string>
 }
 
+const transformBody = (contentType: string, body?: Record<string, string>): string => {
+  if (body === undefined) {
+    return ''
+  }
+
+  if (contentType.toLowerCase().startsWith('application/json')) {
+    return JSON.stringify(body)
+  }
+
+  if (contentType.toLowerCase().startsWith('application/x-www-form-urlencoded')) {
+    let transformedBody = ''
+    Object.entries(body).forEach((entry) => {
+      if (transformedBody.length > 0) {
+        transformedBody += '&'
+      }
+      transformedBody += `${entry[0]}=${entry[1]}`
```

Comment:
Absolutely.  That should have been there already.  I will make this change.

Reply:
I change the mapper to use the URLSearchParams, but I am still returning a string via toString() since this is encapsulated in a transformBody function.